contracts/contracts/GrantRounds.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.5;
+
+contract GrantRounds {
+  // Grant Round ID - The hash of the grant round metadata
+  bytes32 public grantRoundsId;
+  // Metadata Pointer
+  string public metaPtr;
+
+  // Owner
+  address owner;
+
+  modifier onlyOwner() {
+    require(msg.sender == owner);
+    _;
+  }
+
+  constructor(
+    address _owner,
+    bytes32 _grantRoundsId,
+    string memory _metaPtr
+  ) {
+    owner = _owner;
+    grantRoundsId = _grantRoundsId;
+    metaPtr = _metaPtr;
+  }
+
+  // State
+  enum State {
+    CONFIG,
+    ACTIVE,
+    CLAIM,
+    CLOSED
+  }
+
+  // Events
+  // When a Grant Round is created
+  // When a donation is made to a Grant
+
+  uint256 public grantRoundCount = 0;
```

Comment:
Hmm, is the idea that a `GrantRounds` contract would enable more than one round? This might explain the (seemingly) awkward naming I mentioned above.

I would question why we'd want a singleton contract that hosts multiple rounds. I can see this adding a lot of complexity. Couldn't/wouldn't each `GrantsRound` be deployed as it's own contract (possibly with a factory and minimal proxies if desired)?